src/windows.rs

```diff
@@ -4,16 +4,18 @@ use crate::ResolveError;
 
 /// When applicable, converts a [DOS device path](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats#dos-device-paths)
 /// to a normal path (usually, "Traditional DOS paths" or "UNC path") that can be consumed by the `import`/`require` syntax of Node.js.
-/// Returns `None` if the path cannot be represented as a normal path.
-pub fn try_strip_windows_prefix(path: PathBuf) -> Result<PathBuf, ResolveError> {
+///
+/// # Errors
+/// Returns error of [`ResolveError::PathNotSupported`] kind if the path cannot be represented as a normal path.
+pub fn strip_windows_prefix(path: PathBuf) -> Result<PathBuf, ResolveError> {
```

Comment:
Is the renaming intented?

Reply:
Yes. It's named `strip_windows_prefix` before my changes. I've renamed it into `try_strip_windows_prefix` in prior changes as the function has been changed to return `Option<PathBuf>`.

Now considering this function can return `Err(ResolveError)`, I'm restoring the function name in this PR.

----

I was following a traditional rule mentioned here: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/exceptions-and-performance#try-parse-pattern

> ✔️ DO use the prefix "Try" and Boolean return type for methods implementing this (Try-Parse Pattern) pattern.

I'm taking `Err` in Rust the same position as `Exception` in other languages.